cmd/istioctl/inject.go

```diff
@@ -100,7 +100,7 @@ Example usage:
 
 			mesh, err := cmd.GetMeshConfig(client.GetKubernetesClient(), namespace, meshConfig)
 			if err != nil {
-				return err
+				return fmt.Errorf("Please install Istio system into kubernetes before running kube-inject: %v", err)
```

Comment:
The error message should be
`could not find config map   ${namespace}/${meshConfig}, is Istio system installed in ${namespace} ?`